python_modules/dagster/dagster/_core/storage/event_log/sqlite/sqlite_event_log.py

```diff
@@ -409,6 +409,10 @@ def alembic_version(self):
         with self.index_connection() as conn:
             return check_alembic_revision(alembic_config, conn)
 
+    @property
+    def is_sharded(self):
```

Comment:
there are multiple ways it could theoretically be sharded - I think it would be clearer to call this `is_run_sharded`